software/overview.md

```diff
@@ -4,7 +4,7 @@ sort: 1
 
 # Overview
 
-The TurtleBot 4 runs on Ubuntu 20.04 LTS (Focal Fossa) and currently only supports [ROS 2 Galactic](https://docs.ros.org/en/galactic/index.html). The TurtleBot 4 software is entirely open source under the Apache 2.0 license, and is available on the [TurtleBot Github](https://github.com/turtlebot).
+The TurtleBot 4 runs on Ubuntu 20.04 LTS (Focal Fossa) or Ubuntu 22.04 (Jammy Jellyfish) and currently only supports [ROS 2 Galactic](https://docs.ros.org/en/galactic/index.html) or [ROS 2 Humble](https://docs.ros.org/en/humble/index.html) respectively. The TurtleBot 4 software is entirely open source under the Apache 2.0 license, and is available on the [TurtleBot Github](https://github.com/turtlebot).
 
 There are 2 main computers that run software used by TurtleBot 4: the onboard Raspberry Pi 4, the Create® 3 onboard processor. The user can also connect to the robot with their own PC to visualise sensor data, configure the robot, and more. Each computer is required to run Ubuntu 20.04 with ROS 2 Galactic. 
```

Comment:
Replace "Each computer is required to run Ubuntu 20.04 with ROS 2 Galactic." with "Each computer is required to run the same version of Ubuntu and ROS 2 as the TurtleBot 4".